string location_name
geometry_msgs/Pose pose

